### PR TITLE
Dockerfile: disable CGO

### LINKED ISSUE #250: statically link the s5cmd binary
We use your s5cmd utility along with the kubectl utility to perform backups/restores of object stores provisioned by Rook Ceph in our Kubernetes cluster. The object buckets are represented by Object Bucket Claims and Object Buckets. To implement this, we need to have a container with the `s5cmd` utility, the `kubectl` utility, and `bash` to script the various tasks. Presently, the s5cmd contains a couple of dynamic library references which make this a little more complex:
```
ldd s5cmd
    /lib/ld-musl-x86_64.so.1 (0x7f9f5df96000)
    libc.musl-x86_64.so.1 => /lib/ld-musl-x86_64.so.1 (0x7f9f5df96000)
```
Would you consider statically linking the s5cmd executable so that it can be more easily copied to a new container and combined with other utilities as descrbed?

### PR DESCRIPTION
**Pull request by seruman**
Fixes #250

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM golang:1.14-alpine as build
 COPY . /s5cmd/
```
Comment (igungor) [inline Dockerfile:2]: should we bump go image to 1.15?

Comment (seruman) [reply Dockerfile:2]: bumped

**Review by igungor — APPROVED** [review]

**Review by igungor — COMMENTED** [review]
please update changelog.

**Comment by seruman** [comment]
@igungor updated changelog

**Review by aykutfarsak — APPROVED** [review]

**Pull request merged**